gateway/src/apicast/policy/token_introspection/apicast-policy.json

```diff
@@ -33,6 +26,43 @@
         "minimum": 0,
         "maximum": 10000
       }
+    },
+    "required": [
+      "auth_type"
+    ],
+    "dependencies": {
+      "auth_type": {
+        "oneOf": [{
+          "properties": {
+            "auth_type": {
+              "describe": "Get the client and the introspection end point from the oidc issuer endpoint",
```

Comment:
I think this could be improved a bit.

> Use the Client credentials and the OIDC Issuer Endpoint from the OpenID Connect Issuer setting.

Still not perfect, but we can improve that later.

Reply:
Changed.